src/main/java/org/opensearch/flowframework/workflow/WorkflowProcessSorter.java

```diff
@@ -244,11 +254,17 @@ public void validateGraph(List<ProcessNode> processNodes, WorkflowValidator vali
                 );
             }
         }
+    }
 
+    private WorkflowValidator readWorkflowValidator() throws IOException {
+        return WorkflowValidator.parse("mappings/workflow-steps.json");
```

Comment:
Let's handle the exception here (and rethrow an unchecked FlowFrameworkException) here rather than throwing a checked IOException that needs handling far from where the issue happens.

Let's also make this package-private and write a test method to validate this parsing so a user. will never see this error. 🤞

Reply:
Added a new test case to cover all possible ways we parse this timeout value.